javascript/tagAutocomplete.js

```diff
@@ -352,16 +352,16 @@ function addResultsToList(textArea, results, tagword, resetList) {
 
         let displayText = "";
         // If the tag matches the tagword, we don't need to display the alias
-        if (result[3] && !result[0].includes(tagword)) { // Alias
+        if (result[3] && !result[0].search("(^|[^a-zA-Z])" + tagword)>-1) { // Alias
```

Comment:
```suggestion
        if (result[3] && !result[0].includes(tagword)) { // Alias
```
Not needed here, we already filtered our set before, and it will lead to false alias detection.